UPGRADE-5.4.md

```diff
@@ -78,8 +86,8 @@ Monolog
 Notifier
 --------
 
- * [BC BREAK] The following data providers for `TransportTestCase` are now static: `toStringProvider()`, `supportedMessagesProvider()` and `unsupportedMessagesProvider()`
- * [BC BREAK] The `TransportTestCase::createTransport()` method is now static
```

Comment:
why removing?

Reply:
I found these occurrences only in Changelog files, but not in Upgrade files, that's why.

Add it back?

Reply:
I think we should add the prefix in all files yes, to make this clear